### ISSUE TITLE
Overriding Official/Curated repository with Environment variable

### ISSUE THREAD

**Issue by anatawa12** [anatawa12/vrc-get#268]
Since VRChat releases VRCSDK in an unreliable way, VRChat official repository is not reliable for some people. Therefore, there is some idea that its reasonable to create alternative repository to distribute VRCSDK. I agree with that VRChat is not reliable so I want to support this idea with supporting alternative VRCSDK repository.

When the de-facto standard alternative repository become real, I think simple config (or environment variable) but currently there are no so supporting overriding Official repository is good way I think.

**Comment by anatawa12** [comment]
officualを上書きの形で実装したけど上書きではなくOficcialを無視するってことでも良さそ

**Comment by anatawa12** [comment]
vrc-get specific global configurationで上書きしてもいいな

**Comment by Misaka-L** [comment]
I'm a maintainer of a vpm mirror, and it's hard to teach new users who have network issues to overwrite the default official/curated repository url without gui.

So I would like to ask if there is any plan to implement this feature in gui? thanks!

- https://vcc.vrczh.org/repos/official
- https://vcc.vrczh.org/repos/curated
- Our vpm mirror backend source code: https://github.com/vrcd-community/VPMReposSynchronizer

**Comment by Spokeek** [comment]
Maybe it would be time to create a consortium of community tool creators and create our own curated list, showing how we approve new packages and make a proper list that can evolve with the ecosystem.

**Comment by Misaka-L** [comment]
> Maybe it would be time to create a consortium of community tool creators and create our own curated list, showing how we approve new packages and make a proper list that can evolve with the ecosystem.

I agree. The official curated repo have not update for a long long time since it appear, and only provide serval packages.

But the I have some questions
- Who will collect packages and make a repo? 
- How to hosting these packages?
- What is the standard of "curated"?

And I think the community curated repo should be a spreated repo instead of replace the official curated repo.

**Comment by anatawa12** [comment]
I already have issue for vrc-get-curated repository: #294
